spec/parity/environment_spec.rb

```diff
@@ -323,4 +344,14 @@ def stub_migration_path_check(result)
 
     path_stub
   end
+
+  def stub_git_remote(base_name: "parity", environment: "staging")
+    git_remote = double(
```

Comment:
Nice, that was great. The test was valid, but that forced me to specify that `Git` was actually `Git::Base`.